src/components/SearchBar.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { filterByName } from "../actions/index";
+
+class SearchBar extends Component {
+    filterByInput(filter) {
+        this.props.dispatch(filterByName({ value: filter }));
+    }
+    render() {
+        return (
+            <input
+                type="text"
+                className="filter-input"
+                onChange={(e) => this.filterByInput(e.target.value)}
```

Comment:
Maybe you should debounce the input?

I've found an interesting bug - every time the search value changes, it triggers a recalculation of the height for each item on the page, but the height gets bigger every time - see [video](https://www.loom.com/share/7976aa79798349de8c7d16f3d21c9ac9)